example/scripts/generate_ingest/sample_collection/MOD09GQ.006.json

```diff
@@ -4,7 +4,7 @@
 	"process": "modis",
 	"url_path": "{cmrMetadata.Granule.Collection.ShortName}___{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, 0, 3)}",
 	"duplicateHandling": "replace",
-	"granuleId": "^MOD09GQ\\.A[\\d]{7}\\..{6}\\.006\\.[\\d]{13}$",
+	"granuleId": "^MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}$",
```

Comment:
Using all symbols, while increasing possible file counts, was unneeded and was resulting in bad S3 copy behavior.